llama_index/llms/bedrock.py

```diff
@@ -62,6 +65,7 @@ def __init__(
         aws_access_key_id: Optional[str] = None,
         aws_secret_access_key: Optional[str] = None,
         aws_session_token: Optional[str] = None,
+        aws_region_name: Optional[str] = None,
```

Comment:
nitpick, should we add a default region here?

Reply:
I don't think that's a good idea. If the user does have AWS CLI configured and has a default region set, this default aws_region_name would always override it when passing in keys rather than specifying profile name. I think a clear error, is better in this case.

Reply:
thanks Make sense!